packages/svelte/src/lib/component/Uploader.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts" generics="TRouter extends FileRouter">
+  // This eslint comment is because eslint-plugin-svelte does not support component generics as of yet:
```

Comment:
This is incredibly sad but I appreciate you for calling it out

Reply:
Yeah it's a bummer, but I did find a workaround for it :)